bin/strings-import.js

```diff
@@ -2,74 +2,74 @@
 "use strict";
 
 /* eslint-disable no-console */
+const fetch = require("node-fetch");
+
+/* globals cd, ls, mkdir, rm, ShellString */
+require("shelljs/global");
 
 const DEFAULT_LOCALE = "en-US";
+const L10N_CENTRAL = "https://hg.mozilla.org/l10n-central";
+const PROPERTIES_PATH = "raw-file/default/browser/chrome/browser/activity-stream/newtab.properties";
+const STRINGS_FILE = "strings.properties";
 
-/* globals cd, cp, ls, mkdir, rm */
-require("shelljs/global");
+// Get all the locales in l10n-central
+async function getLocales() {
+  console.log(`Getting locales from ${L10N_CENTRAL}`);
 
-const args = process.argv.slice(2);
-if (args.length < 1) {
-  throw Error("Please provide the path to the target strings repository");
+  // Add all non-test sub repository locales
+  const locales = [];
+  const subrepos = await (await fetch(`${L10N_CENTRAL}?style=json`)).json();
+  subrepos.entries.forEach(({name}) => {
+    if (name !== "x-testing") {
+      locales.push(name);
+    }
+  });
+
+  console.log(`Got ${locales.length} locales: ${locales}`);
+  return locales;
 }
 
-// Use the first script argument as the source of localized strings
-const [stringsPath] = args;
-const stringsRepo = require("simple-git")(stringsPath);
+// Save the properties file to the locale's directory
+async function saveProperties(locale) {
+  // Only save a file if the repository has the file
+  const url = `${L10N_CENTRAL}/${locale}/${PROPERTIES_PATH}`;
+  const response = await fetch(url);
+  if (response.status !== 200) {
```

Comment:
`if (!response.ok)` might be a little less restrictive

Reply:
ok? ok!? We're not ok! Must be 200!!! ok… ok.

Reply:
Updated ;)